_component_design/buttons.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Buttons
-parent: Editing
+parent: Inputs and Controls
```

Comment:
If you change the parent, you need to change the secondary so that it appears under each category. The current categories are from the IA sheet. If there's no secondary category, remove the metadata key.

Reply:
Removed `secondary` metadata